src/server/http/__tests__/index.js

```diff
@@ -1,17 +1,26 @@
 import expect from 'expect.js';
 import * as kbnTestServer from '../../../test_utils/kbn_server';
+import { esTestCluster } from '../../../test_utils/es';
 
 describe('routes', function () {
-  this.slow(10000);
-  this.timeout(60000);
+  this.slow(60 * 1000);
+  this.timeout(3 * 60 * 1000);
```

Comment:
Rather than hardcode these times all over, and since they are directly related to the behaviors of esTestCluster, how about moving them to a constant from test_utils/es?